### PR TITLE
Wavefront configuration docs

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves https://github.com/spring-cloud/spring-cloud-dataflow/issues/4016

### CODE REVIEW

**Comment by ilayaperumalg** [comment]
LGTM

File: content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md [hunk 1]
```diff
@@ -201,6 +203,118 @@ grafana-info:
...
+
+To mount the secret and make it available to Spring Cloud Data Flow, modify the file `src/kubernetes/server/server-deployment.yaml` making the following additions:
+
+```yaml
+spec:
+  containers:
+    - name: scdf-server
+      volumeMounts:
+        - name: wavefront-api
+          mountPath: /etc/secrets/wavefront-api
```
Comment (ilayaperumalg) [inline content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md:240]: @chrisjs I realized that we need a note saying this mount path needs to belong to whatever `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` is configured to. Isn't it?

Comment (chrisjs) [reply content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md:240]: i don't think its needed as one can see it already but if you have some wording you would like to add im open to it

Comment (ilayaperumalg) [reply content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md:240]: I thought the `mountPath` has to be in the paths that `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` is configured. I could only make it to work when I modified the `mountPath` to something that `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` has it.

Comment (chrisjs) [reply content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md:240]: in `server-deployment.yaml` we have:

```
        - name: SPRING_CLOUD_KUBERNETES_SECRETS_PATHS
          value: /etc/secrets
```

?

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Pull request merged**